tests/unittests/Foundation/NSPointerArrayTests.mm

```diff
@@ -61,9 +55,12 @@ @implementation NSPointerArraySubClass
     ASSERT_EQ_MSG(array.count, 3, @"should have 3 items in the array.");
 }
 
-TEST(NSPointerArray, Subclassing) {
+// Note: On the reference platform when a class that does not support sub-classing is sub-classed, the parent class is
```

Comment:
Discussed offline - we can keep this as OSX_DISABLED

---
In reply to: [73561438](https://github.com/Microsoft/WinObjC/pull/806#discussion_r73561438) [](ancestors = 73561438,73437159)